debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/ParallelSnapshotReader.java

```diff
@@ -177,17 +203,31 @@ public String name() {
 
         @Override
         public boolean test(SourceRecord ourSourceRecord) {
+            logger.info("TESTING PARALLEL HALTING PREDICATE: {}", ourSourceRecord.sourceOffset().toString());
             // we assume if we ever end up near the end of the binlog, then we will remain there.
             if (!thisReaderNearEnd.get()) {
+                //ourSourceRecord.value()
+                Long sourceRecordTimestamp = (Long) ourSourceRecord.sourceOffset().get(SourceInfo.TIMESTAMP_KEY);
+                if (sourceRecordTimestamp == null) {
+                    // TODO timestamp is null because it's not the -real- offset!!
+                    // it's the fake, previous offset from the other reader
+                    // in this case, it's from a snapshot. So that's why it has no timestamp.
+                    logger.info ("TIMESTAMP IS NULL! (FOR SOME REASON?!)");
```

Comment:
Hum, I don't think we should log messages like this. What would a user of the connector do with this information?